sky/tools/gn

```diff
@@ -44,6 +48,7 @@ def to_gn_args(args):
 
     gn_args['is_debug'] = args.debug
     gn_args['is_clang'] = args.clang and args.target_os not in ['android', 'fnl']
+    gn_args['flutter_develop_mode'] = args.develop
```

Comment:
Develop mode is true by default and will be false if you run gn with the --deploy flag.  Dart product mode is on if develop mode is off.